Tweaks/TweaksAssembly/BombWrapper.cs

```diff
@@ -234,43 +234,157 @@ public void Awake()
 						new TTKComponentSolver(component, bombModule, Tweaks.CurrentMode.Equals(Mode.Zen) ? Modes.initialTime : timerComponent.TimeRemaining);
 						break;
 
-					// Correct some mispositioned objects in older modules
+					// Correct some mispositioned objects in some regular modules
 					case "ForeignExchangeRates":
 					case "resistors":
 					case "CryptModule":
 					case "LEDEnc":
 						// This fixes the position of the module itself (but keeps the status light in its original location, which fixes it)
 						component.transform.Find("Model").transform.localPosition = new Vector3(0.004f, 0, 0);
 						break;
+					case "ColourFlash":
+					case "ColourFlashES":
+					case "ColourFlashPL":
+						// This fixes the position of the module itself (but keeps the status light in its original location)
+						component.transform.Find("ModuleBackground").transform.localPosition = Vector3.zero;
+						// This fixes the position of the status light
+						component.GetComponentInChildren<StatusLightParent>().transform.localPosition = new Vector3(0.075167f, 0.01986f, 0.076057f);
+						break;
 					case "Listening":
 						// This fixes the Y-coordinate of the position of the status light
 						component.transform.Find("StatusLight").transform.localPosition = new Vector3(-0.0761f, 0.01986f, 0.075f);
 						break;
 					case "TwoBits":
 					case "errorCodes":
 					case "primeEncryption":
-					case "memorableButtons":
 					case "babaIsWho":
+					case "combinationLock":
+						// This fixes the position of the status light
+						component.GetComponentInChildren<StatusLightParent>().transform.localPosition = new Vector3(0.075167f, 0.01986f, 0.076057f);
+						break;
+					case "matrix":
+						// This fixes the position of the module itself
+						for (int i = 0; i < component.transform.childCount; i++)
+							component.transform.GetChild(i).transform.localPosition -= new Vector3(0, 0.0053061005f, 0);
```

Comment:
There's a better way to iterate over children:
```suggestion
						foreach (Transform child in component.transform)
							child.localPosition -= new Vector3(0, 0.0053061005f, 0);
```